authorization/permission/model/permission_model_service.go

```diff
@@ -27,11 +27,15 @@ func NewPermissionModelService(db *gorm.DB, repo repository.Repositories) Permis
 	}
 }
 
-// Creates a new organization.  The specified identityID is the user creating the organization, while the name parameter
-// specifies the organization name.  The organization's identity ID is returned.
-func (s *GormPermissionModelService) HasScope(ctx context.Context, identityID uuid.UUID, resourceID string, scope string) (bool, error) {
+// The HasScope method does a permission check for a user, to determine whether they have a particular scope for the
```

Comment:
Good description! But a function description in Go should start with the function name. It's go convention.
`// HasScope does a permission check...`